### DISCUSSION TITLE
Is there a plan to support KIP-48 Delegation token support for Kafka？

### DISCUSSION THREAD

**Discussion by sailing27** [edenhill/librdkafka#4478] [Q&A]
Is there a plan to support KIP-48 Delegation token support for Kafka？

**Comment by Chrisss93** [comment]
I'd be interested in this as well!  Delegation tokens are strangely a pretty old feature in Kafka but also a scarcely supported one.  As far as I know, only the native java client has full support for it (meaning both admin-client APIs AND token authentication via pseudo-SCRAM).  There's also [this project](https://github.com/Kong/lua-resty-kafka) which at least allows token authentication in lua.
